packages/schemas/alterations/next-1709263626-regenerate-dau-data.ts

```diff
@@ -0,0 +1,45 @@
+import { generateStandardId } from '@logto/shared/universal';
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+type ActiveUserInteractionLog = {
+  tenantId: string;
+  userId: string;
+  createdAt: number;
+};
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    // Delete all record from `daily_active_users` table
+    await pool.query(sql`delete from daily_active_users;`);
+
+    // Retrieve all active user logs from `logs` table
+    const { rows: interactionLogs } = await pool.query<ActiveUserInteractionLog>(sql`
```

Comment:
Not sure if Github Actions allow such large amount of data in memory, maybe should split into sub jobs.

Reply:
Per offline discussion, It's ok to execute this script since our data is not large.